### DISCUSSION TITLE
go 二进制启动流程分析

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#1] [General]
# go 二进制启动流程分析

借助 gdb 调试信息观测 golang 二进制启动的过程，从侧面理解 go runtime 的实现细节以及 go 二进制的内存布局设计。

https://www.edony.ink/details-of-go-binary-launch/

**Comment by edonyzpc** [comment]
依托于 Github 的讨论去建设完成啦！欢迎大家来讨论

**Reply by edonyzpc** [reply]
- 支持嵌入式评论
- 支持 `Markdown`
